packages/rancher-monitoring/generated-changes/patch/templates/exporters/node-exporter/servicemonitor.yaml.patch

```diff
@@ -0,0 +1,8 @@
+--- charts-original/templates/exporters/node-exporter/servicemonitor.yaml
++++ charts/templates/exporters/node-exporter/servicemonitor.yaml
+@@ -1,4 +1,4 @@
+-{{- if .Values.nodeExporter.enabled }}
++{{- if (and .Values.nodeExporter.enabled (not .Values.hardenedNodeExporter.enabled)) }}
```

Comment:
Instead of this, let's fail if the user has both enabled.
```
{{- if .Values.nodeExporter.enabled }} 
{{- if .Values.hardenedNodeExporter.enabled }}
{{- required "Cannot set both .Values.nodeExporter.enabled and .Values.hardenedNodeExporter.enabled to true" "" }}
{{- end }}
```